BlueprintUI/Sources/Layout/LayoutWriter.swift

```diff
@@ -49,7 +49,10 @@ public struct LayoutWriter : Element {
     public var content: ElementContent {
         ElementContent { size, env in
             var builder = Builder()
-            self.build(Context(size: size), &builder)
+            
+            if size.maximum.width > 0 && size.maximum.height > 0 {
```

Comment:
You should probably document this behavior. Most others containers will still include their children even if they're laying out in an empty frame. This change means a zero frame won't even create any children.